components/style/properties/helpers.mako.rs

```diff
@@ -903,6 +903,7 @@
     <%call expr="longhand(name,
                           predefined_type=length_type,
                           logical=logical,
+                          is_size_type=True,
```

Comment:
Let's rename this to something that screams out more, like `is_gecko_size_type_hack` or something.